packages/manager/apps/dedicated/client/app/account/user/users/sso/details/user-users-sso-details.html

```diff
@@ -0,0 +1,64 @@
+<oui-header heading="{{:: 'user_users_sso_detail_title' | translate }}">
+</oui-header>
+<div>
+    <a
+        class="d-inline-block col-md-12 mb-12"
```

Comment:
**question:** Why mixing inline-block helper and `col-*`? (Parent `col-` element must be a `row` but I'm wondering why a `col` is needed here.

**nitpick(blocking):** `mb-12` helper doesn't exist.